### ISSUE TITLE
Computer configuration

### ISSUE THREAD

**Issue by Legend147** [HewlettPackard/quartz#4]
My Computer configuration is very common personal computer. Intel core i3, Ubuntu 14.04. Can I  install quartz successfully

**Comment by hvolos** [comment]
It should be possible as long as your core i3 is one of Ivybridge, Haswell, Sandybridge.

**Comment by Legend147** [comment]
Thank you very much. Is Write non volatile latency not yet implemented？Are there some good approach to emulate the write latency?

**Comment by sarsanaee** [comment]
@hvolos But I guess it is not possible because the CPUs should be Xeon.

**Comment by Legend147** [comment]
when I run the test, I meet the ERROR : No supported processor found

**Comment by sarsanaee** [comment]
@Legend147 Yeah you are right. In the paper, it is mentioned that this tool is developed for Xeon-based systems. You could check out `src/lib/cpu/cpu.c#L138` in order to find out how the CPU models are verified.

**Comment by hvolos** [comment]
@Legend147: Our performance model uses the cycles_activity performance counter to estimate the cycles a processor stalls waiting for memory (http://oprofile.sourceforge.net/docs/intel-ivybridge-events.php). If your processor supports such a counter, then it would be possible to extend Quartz as sarsanaee explains.

Concerning modelling writes to persistent memory: our primary focus was modelling reads, that require taking into account the processor cache that may hide the latency to NVM. For writes, we assume one would model them using previous techniques like introducing a spin delay when flushing the write to NVM. We didn't include that code, but we're planning to do so in the next release (when we get the chance)

**Comment by HaotianChen616** [comment]
How about Kaby lake? Is quartz support more types of cpu now?
